src/containers/Passphrase.jsx

```diff
@@ -0,0 +1,64 @@
+import { connect } from 'react-redux'
+import get from 'lodash/get'
+import compose from 'lodash/flowRight'
+import Alerter from 'cozy-ui/react/Alerter'
+import { translate } from 'cozy-ui/react/I18n'
+import { AUTH_MODE } from 'actions/twoFactor'
+import {
+  updatePassphrase,
+  updatePassphrase2FAFirst,
+  updatePassphrase2FASecond
+} from 'actions/passphrase'
+import { fetchInfos } from 'actions'
+import PassphraseView from 'components/PassphraseView'
+
+const mapStateToProps = state => ({
+  fields: state.fields,
+  isTwoFactorEnabled:
+    get(state, 'fields.auth_mode.value') === AUTH_MODE.TWO_FA_MAIL,
+  instance: state.instance,
+  passphrase: state.passphrase
+})
+
+const mapDispatchToProps = (dispatch, ownProps) => ({
+  onPassphraseSimpleSubmit: (current, newVal) => {
+    return (
+      dispatch(updatePassphrase(current, newVal))
+        .then(() => Alerter.info(ownProps.t('ProfileView.password.reload')))
+        // eslint-disable-next-line no-console
+        .catch(e => console.error(e))
+    )
+  },
+  onPassphrase2FAStep1: current => {
+    return dispatch(updatePassphrase2FAFirst(current)).catch(e =>
+      // eslint-disable-next-line no-console
+      console.error(e)
+    )
+  },
+  onPassphrase2FAStep2: (current, newVal, twoFactorCode, twoFactorToken) => {
+    return (
+      dispatch(
+        updatePassphrase2FASecond(
+          current,
+          newVal,
+          twoFactorCode,
+          twoFactorToken
+        )
+      )
+        .then(() => Alerter.info(ownProps.t('ProfileView.password.reload')))
+        // eslint-disable-next-line no-console
+        .catch(e => console.error(e))
+    )
+  },
+  fetchInfos: () => dispatch(fetchInfos())
+})
+
+const Password = compose(
+  translate(),
+  connect(
+    mapStateToProps,
+    mapDispatchToProps
+  )
+)(PassphraseView)
```

Comment:
Ah OK, I understand better my question on the fetchInfos from the parent. I think it would be clearer if all the dispatch functions and the connect were in the PassphraseView component.